voting/client-web/src/components/Header/Header.tsx

```diff
@@ -29,6 +30,20 @@ const HeaderImpl = ({ classes, isReadOnly }) => {
     { label: t('Rewards'), url: '/reward' },
   ];
 
+  function displayInMetamask() {
```

Comment:
Need to ask you about our work practices, do we want all effects to be redux based or smth ?

Reply:
Hedron isn't using any state management library. Depending on the use case we can decided on the proper solution. If we'll see that Hedron is getting too complicated regarding state, we'll choose a library.